server/routerlicious/packages/lambdas/src/nexus/index.ts

```diff
@@ -1045,67 +1045,65 @@ export function configureWebSocketServices(
 			},
 		);
 
-		// Message sent when a new signal is submitted to the router
-		socket.on(
-			"submitSignal",
-			(clientId: string, contentBatches: (IDocumentMessage | IDocumentMessage[])[]) => {
-				// Verify the user has subscription to the room.
-				const room = roomMap.get(clientId);
-				if (!room) {
+		/**
+		 * Message sent when a new signal is submitted to the router.
+		 * @see {@link DocumentDeltaConnection.emitMessages} for client-side emission of signals.
```

Comment:
I don't think that link goes anywhere. DocumentDeltaConnection is not something that exists in this scope.
You can open the source on GitHub, go to the relevant lines, and get a permalink.